<launch>
    <param name="use_sim_time" value="true" />
    <rosparam file="$(find local_map_builder)/config/config.yaml" />
    <node name="local_costmap" pkg="local_map_builder" type="local_costmap" respawn="true" output="screen" />
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find local_map_builder)/rviz/rviz_config.rviz" />
</launch>
